### PR TITLE
[WIP]: Cypress automation for Trace analytics dashboard application

### PR DESCRIPTION
**Pull request by pratibhapandey16**
### Description
Automated different flows under trace analytics application

### Check List
- [ ] Trace analytics dashboard
- [ ] Traces 
- [ ] Services
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

@spattnaik @abasatwar

### CODE REVIEW

File: dashboards-observability/.cypress/integration/5_trace_analytics_services.spec.js [hunk 1]
```diff
@@ -115,9 +137,144 @@ describe('Testing service view', () => {
...
+  beforeEach(() => {
+    cy.visit('app/observability-dashboards#/trace_analytics/services', {
+      onBeforeLoad: (win) => {
+        win.sessionStorage.clear();
+      },
+    });
+    setTimeFilter();
+  });
+
+  it('Renders the spans table and verify coloums headers', () => {
```
Comment (mengweieric) [inline dashboards-observability/.cypress/integration/5_trace_analytics_services.spec.js:179]: Do you mean 'column'?

File: dashboards-observability/.cypress/integration/6_trace_analytics_traces.spec.js [hunk 2]
```diff
@@ -113,3 +113,44 @@ describe('Testing trace view', () => {
...
+    cy.go('back');
+    cy.wait(delay);
+    cy.get('.euiButtonEmpty__text').contains('Rows per page').click();
+    cy.get('.euiContextMenuItem__text').contains('15 rows').click();
+    cy.get('.euiTable--auto')
+    .find("tr")
+    .then((row) => {
+      //row.length will give you the row count
+      cy.log(row.length-1);
+      let total=row.length-1;
```
Comment (joshuali925) [inline dashboards-observability/.cypress/integration/6_trace_analytics_traces.spec.js:152]: is this needed?

**Review by joshuali925 — APPROVED** [review]

File: dashboards-observability/.cypress/integration/5_trace_analytics_services.spec.js (hunk 1 above)
Comment (nidhisinghai) [reply dashboards-observability/.cypress/integration/5_trace_analytics_services.spec.js:179]: Yes, updated the Spelling for same

File: dashboards-observability/.cypress/integration/6_trace_analytics_traces.spec.js (hunk 2 above)
Comment (nidhisinghai) [reply dashboards-observability/.cypress/integration/6_trace_analytics_traces.spec.js:152]: Updated this code for assert

**Review by mengweieric — APPROVED** [review]

**Review by joshuali925 — APPROVED** [review]

**Review by eugenesk24 — APPROVED** [review]
Looks good!

**Pull request merged**